src/main/java/com/sandvoxel/generitech/common/tileentities/machines/TileEntityPulverizer.java

```diff
@@ -201,98 +195,119 @@ public boolean canBeRotated() {
         return true;
     }
 
-    public long getPower() {
-        return container.getStoredPower();
+
+
+    public boolean canWork(){
+        if (machineTier == MachineTier.TIER_0){
+            return fuelRemaining > 0;
+        }else  {
+         return container.getStoredPower() >= powerUsage;
+        }
     }
 
+    public boolean cancrush(ItemStack item){
+            return PulverizerRegistry.containsInput(item);
+    }
+
+    public void burnTime(int i)
+    {
+        if (fuelRemaining==0){
+            if (inventory.getStackInSlot(4).getItem() == lastFuelType && cancrush(inventory.getStackInSlot(i))) {
+                fuelRemaining = lastFuelValue;
+
+
+            } else if (inventory.getStackInSlot(4).getItem() != lastFuelType && cancrush( inventory.getStackInSlot(i) ) )
+            {
+                fuelRemaining = net.minecraft.tileentity.TileEntityFurnace.getItemBurnTime(inventory.getStackInSlot(4));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3ACommentedOutCodeLine)